### PR TITLE
Documentation cleanup

### PR DESCRIPTION
**Pull request by phayes**

### CODE REVIEW

File: src/DOMDoc.php [hunk 1]
```diff
@@ -16,50 +16,48 @@ class DOMDoc extends \DOMDocument {
...
-   *  Auto-register namespaces. All namespaces in the root element will be registered for use in xpath queries.
-   *  Namespaces that are not declared in the root element will not be auto-registered
-   *  Defaults to TRUE (Meaning it will auto register all auxiliary namespaces but not the default namespace).
-   *  Pass a prefix string to automatically register the default namespace.
-   *  Pass FALSE to disable auto-namespace registeration
-   * 
-   * @param bool $error_checking
+   * @param object|string $xml
+   *   $xml can either be an XML string, a DOMDocument, or a DOMElement.
+   *   You can also pass FALSE or NULL (or omit it) and load XML later using loadXML or loadHTML.
```
Comment (pwaterz) [inline src/DOMDoc.php]: if null or false is allowed, that should be part of @param

File: src/DOMDoc.php [hunk 2]
```diff
@@ -16,50 +16,48 @@ class DOMDoc extends \DOMDocument {
...
+   * @param true|string $auto_register_namespaces
```
Comment (pwaterz) [inline src/DOMDoc.php]: bool|string

File: src/DOMDoc.php [hunk 3]
```diff
@@ -16,50 +16,48 @@ class DOMDoc extends \DOMDocument {
...
+   * @param object|string $xml
+   *   $xml can either be an XML string, a DOMDocument, or a DOMElement.
+   *   You can also pass FALSE or NULL (or omit it) and load XML later using loadXML or loadHTML.
+   * @param true|string $auto_register_namespaces 
+   *   Auto-register namespaces. All namespaces in the root element will be registered for use in xpath queries.
+   *   Namespaces that are not declared in the root element will not be auto-registered
+   *   Defaults to TRUE (Meaning it will auto register all auxiliary namespaces but not the default namespace).
+   *   Pass a prefix string to automatically register the default namespace.
+   *   Pass FALSE to disable auto-namespace registeration.
+   * @param false|string $error_checking
```
Comment (pwaterz) [inline src/DOMDoc.php]: bool|string

File: src/DOMDoc.php [hunk 4]
```diff
@@ -69,46 +67,48 @@ public function __construct($xml = FALSE, $auto_register_namespaces = TRUE, $err
...
+   * Given a namespace prefix, get the URL.
+   *
    * @param string $prefix
-   *  namespace prefix
-   * 
-   * return string|false
-   *  The namespace URL or FALSE if there is no namespace with that prefix
+   *   Namespace prefix.
+   *
+   * @return string|false
```
Comment (pwaterz) [inline src/DOMDoc.php]: string|bool

File: src/DOMDoc.php [hunk 5]
```diff
@@ -121,24 +121,24 @@ public function lookupURL($prefix) {
...
    * @param string $xpath
-   *  xpath to be used for query
-   * 
+   *   XPath to be used for query.
+   *
    * @param mixed $context
-   *  $context can either be an xpath string, or a DOMElement
-   *  Provides context for the xpath query
-   * 
+   *   $context can either be an xpath string, or a DOMElement.
```
Comment (pwaterz) [inline src/DOMDoc.php]: $context shouldn't be in the comment

File: src/DOMDoc.php (hunk 4 above)
Comment (phayes) [reply src/DOMDoc.php]: no, true is never returned

Comment (pwaterz) [reply src/DOMDoc.php]: ahh didn't know that was valid

**Pull request merged**